data/Danny4life.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Segun Osiki",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Fullstack Developer",
+  "avatar": "https://github.com/Danny4life.png",
```

Comment:
As of our latest update, we no longer need the `avatar` key. So please remove this from your file. 

```suggestion
```